packages/concerto-types/scripts/codegen.js

```diff
@@ -4,12 +4,14 @@ const { ModelManager } = require('@accordproject/concerto-core');
 const { CodeGen: { TypescriptVisitor }} = require('@accordproject/concerto-codegen');
 const { FileWriter } = require('@accordproject/concerto-util');
 const path = require('path');
+const DCS_MODEL = require('@accordproject/concerto-core/lib/decoratormanager').DCS_MODEL;
 
 /**
  * Generate TypeScript files from the metamodel.
  */
 async function main() {
     const modelManager = new ModelManager({addMetamodel:true, strict: true});
+    modelManager.addCTOModel(DCS_MODEL, 'decoratorcommands@0.4.0.cto');
```

Comment:
Can you please run `npm run codegen` and also commit the generated type files to this PR?